tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -85,6 +88,11 @@ spec:
         fi
 
         DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        RESULTS_DIR="$(workspaces.data.path)/$(params.resultsDirPath)"
+        RESULTS_FILE="${RESULTS_DIR}/publish-pyxis-repository-results.json"
+        RESULTS_JSON='{"images":[]}'
+
+        mkdir -p "${RESULTS_DIR}"
```

Comment:
You shouldn't need this. It is done here https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/collect-data/collect-data.yaml#L87

Reply:
Fixed, got it.